docs/src/surfaces/card/card.jsx

```diff
@@ -309,22 +313,23 @@ export default class ViewsTitleBar extends React.Component {
                     </Highlighter>
 
                     {/* Nested Card */}
-                    <Header size="large" style={{ marginTop: '55px' }} sub>
+                    <Heading variant="h2">
                         Nested Card
-                        <Header.Subheader>
-                            A card can give the appearance of being nested. The parent's background color needs to be set to
-                            {' '}
-                            <code>color(backgroundColorNest)</code>
-                            .
-                        </Header.Subheader>
-                    </Header>
+                    </Heading>
+
+                    <Typography variant="body1">
+                        A card can give the appearance of being nested. The parent's background color needs to be set to
+                        {' '}
+                        <code>color(backgroundColorNest)</code>
```

Comment:
Is this still useful?  Or should we start talking about theme/palette here?

Reply:
Should talk about theme, eventually.

Reply:
This is non-blocking for now.  Resolving.